File: android/expoview/src/main/java/abi29_0_0/host/exp/exponent/modules/api/components/gesturehandler/BaseGestureHandlerInteractionController.java
package abi29_0_0.host.exp.exponent.modules.api.components.gesturehandler;

public abstract class BaseGestureHandlerInteractionController
        implements GestureHandlerInteractionController {

  @Override
  public boolean shouldWaitForHandlerFailure(GestureHandler handler,
                                             GestureHandler otherHandler) {
    return false;
  }

  @Override
  public boolean shouldRequireHandlerToWaitForFailure(GestureHandler handler,
                                                      GestureHandler otherHandler) {
    return false;
  }

  @Override
  public boolean shouldRecognizeSimultaneously(GestureHandler handler,
                                               GestureHandler otherHandler) {
    return false;
  }
}

File: android/versioned-abis/expoview-abi44_0_0/src/main/java/abi44_0_0/host/exp/exponent/modules/api/components/gesturehandler/BaseGestureHandlerInteractionController.kt
package abi44_0_0.host.exp.exponent.modules.api.components.gesturehandler

abstract class BaseGestureHandlerInteractionController : GestureHandlerInteractionController {
  override fun shouldWaitForHandlerFailure(
    handler: GestureHandler<*>,
    otherHandler: GestureHandler<*>,
  ) = false

  override fun shouldRequireHandlerToWaitForFailure(
    handler: GestureHandler<*>,
    otherHandler: GestureHandler<*>,
  ) = false

  override fun shouldRecognizeSimultaneously(
    handler: GestureHandler<*>,
    otherHandler: GestureHandler<*>,
  ) = false
}
